test/Backends/Michelson.hs

```diff
@@ -629,18 +544,13 @@ identityAppTerm2 =
 --   ,PrimEx (DIG 1)
 --   ,PrimEx (PAIR : @ % %)
 --   ,PrimEx (DIPN 1 [PrimEx DROP])]
-
 identityAppExpr2 :: Term
 identityAppExpr2 =
-  Ann
-    one
-    identityType2
+  Ann one identityType2
     $ J.LamM [] ["x"]
     $ Ann one (primTy (Untyped.pair opl Untyped.unit))
     $ J.AppM
-      ( Ann
-          one
-          (J.Pi one primPairTy (primTy (Untyped.pair unitl Untyped.unit)))
+      ( Ann one (J.Pi one primPairTy (primTy (Untyped.pair unitl Untyped.unit)))
```

Comment:
I prefer the old style for this, as it allowed the arguments to align with the $